doc/managing_policies.md

```diff
@@ -5,30 +5,48 @@ fh-immuta-utils supports creating global policies to limit access to data source
 The intended use-case is where access to sensitive data is granted on the basis of membership in IAM groups,
 where sensitive columns in data sources are tagged as such.
 
-The repo provides a script that can create/remove policies based on provided tagging spec files.
+The repo provides a script that can create, update, and remove policies based on provided policy spec files.
 
-As an example, imagine that the following tag spec file is used for tagging data sources:
+As an example, imagine that the following data policy spec file is used:
 
 ``` yaml
----
-TAG_MAP:
-  ssn:
-    - phi.ssn
-
-TAG_GROUPS:
-  phi.ssn:
-    - "Medical Staff"
-    - "Researchers"
+DATA_POLICIES:
+  policy_1:
+    actions:
+      - type: "masking"  # mask fields
+        description: ""
+        rules:
+          - type: "masking"  # using hashing
+            config:
+              fields:
+                tags: ["tag1"]  # for fields tagged with "tag1"
```

Comment:
the `fields` specify where the masking action will apply. so here, it would be `fields` tagged with `tag1`. in the `cirumstances`, we use `tag1` to identify the data sources where this masking action will apply.

yes, fields and columns are the same.